src/Service.Tests/Snapshots/ConfigurationTests.TestReadingRuntimeConfigForMsSql.verified.txt

```diff
@@ -2095,16 +2095,13 @@
       GetPublisher: {
         Source: {
           Object: get_publisher_by_id,
-          Type: stored-procedure,
-          Parameters: {
-            id: 1
-          }
+          Type: stored-procedure
         },
         GraphQL: {
           Singular: GetPublisher,
           Plural: GetPublishers,
           Enabled: true,
-          Operation: Mutation
+          Operation: Query
```

Comment:
why this change?

Reply:
GetPublisher is a getter method returning results and hence should be marked as query as typically mutations are used to create/update/delete from db.